ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-release-2.2.yaml

```diff
@@ -94,7 +94,7 @@ releases:
   latest:
     release:
       channel: fast
-      version: "4.15"
+      version: "4.17"
```

Comment:
If we bump the release version in 2.2 we should do the same in the `main` branch.